rust/scx_rustland_core/assets/bpf.rs

```diff
@@ -197,7 +197,7 @@ impl<'cb> BpfScheduler<'cb> {
         // Open the BPF prog first for verification.
         let mut skel_builder = BpfSkelBuilder::default();
         skel_builder.obj_builder.debug(debug);
-        let mut skel = scx_ops_open!(skel_builder, open_object, rustland)?;
+        let mut skel = scx_ops_open!(skel_builder, open_object, rustland, None)?;
```

Comment:
Should we init and pass an open_opts object here, so it's used consistently everywhere? It should be also easier to read, `None` is quite obscure here, wdyt?

Reply:
Yeah, that sounds good.